### PR TITLE
Update EasyEffects instructions for 6.x.x

### PR DESCRIPTION
**Pull request by DianaNites**
Updates and clarifies the instructions for PulseEffects/EasyEffects for version 6.x.x, which drastically changed the interface.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,9 +190,16 @@ Viper4Android. It provides a system wide solution, has a convolution engine but
 ### Linux
 #### PulseEffects / EasyEffects
 [PulseEffects / EasyEffects](https://github.com/wwmm/easyeffects) is a Linux module with wide variety of signal
-processing tools including parametric equalizer. From version 5.0.0 onwards, PulseEffects was renamed to EasyEffects and
+processing tools including parametric equalizer.
+  
+From version 6.0.0 onwards, first select the `plugins` tab at
+the bottom of the screen, add the equalizer plugin, and load APO settings by clicking "Load APO Preset" and
+selecting the ParametricEQ.txt file. Pre-amp can be adjusted with the input slider.
```
Comment (bhack) [inline README.md:197]: I think it will be not required in 6.1.4

Comment (wwmm) [reply README.md:197]: It was a recent contribution that will be present in the next release https://github.com/wwmm/easyeffects/pull/1193. But have in mind I was not the one testing it. I do not use APO presets.

Comment (bhack) [reply README.md:197]: /cc @ishitatsuyuki

Comment (ishitatsuyuki) [reply README.md:197]: Yes, it won't be required in 6.1.4.

Comment (DianaNites) [reply README.md:197]: Well, the current version, 6.1.3, does, so..

Comment (bhack) [reply README.md:197]: Can we just claim that for Easyeffects <= 6.1.3 you need to manually adjust the Pre-amp with the input slider ?

Comment (DianaNites) [reply README.md]: Done

**Comment by jaakkopasanen** [comment]
Thanks a lot for the PR! I made the changes manually because I also wanted to add info about convolution and parameteric eq filter types from #340 and #282, respectively.

**Comment by ishitatsuyuki** [comment]
here's my two cents, but it looks like that:
- IIR (minimum-phase) replicates Equalizer APO's behavior, which AutoEQ is targeting
- FIR and FFT calculates a linear-phase version based on the IIR filter's frequency response, the two modes appears to be same except the latter convolves more efficiently
- No idea about SPM

I think IIR is the one to use, since it has minimum latency, is efficient, and does replicate other platform's behavior.

**Comment by bhack** [comment]
Just to mention a reply by oratory1990 :

https://www.reddit.com/r/oratory1990/comments/e7tmzk/iir_and_fir_filtering/

**Pull request closed without merging**